Tools/scripts/build_binaries.py

```diff
@@ -505,9 +505,9 @@ def build_vehicle(self, tag, vehicle, boards, vehicle_binaries_subdir,
                         files_to_copy.append(filepath)
                 if not os.path.exists(bare_path):
                     raise Exception("No elf file?!")
-                # only copy the elf if we don't have other files to copy
-                if len(files_to_copy) == 0:
-                    files_to_copy.append(bare_path)
+
+                # always copy the elf file across:
```

Comment:
need to twiddle for the elf extension, either by making chibios build use a .elf extension or renaming here

Reply:
Done by doing the rename here.